agent/structs/structs.go

```diff
@@ -2438,6 +2478,21 @@ func (r *KeyringResponses) New() interface{} {
 	return new(KeyringResponses)
 }
 
+// String converts message type int to string
+func (m MessageType) String() string {
+	s, ok := requestTypeStrings[m]
+	if ok {
+		return s
+	}
+
+	s, ok = enterpriseRequestType()
```

Comment:
I think `enterpriseRequestType` will need to accept some argument, right? Does it need to be passed `m` ?

Reply:
Good catch. Yes it does.